<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window4</class>
 <widget class="QWidget" name="window4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>节点1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Battery" name="battery1" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>248</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>248</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bat1">
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>节点2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Battery" name="battery2" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>248</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>248</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bat2">
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>节点3：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Battery" name="battery3" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>248</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>248</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bat3">
       <property name="text">
        <string>100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>124</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>124</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/back.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Battery</class>
   <extends>QWidget</extends>
   <header>battery.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
